integration-tests/narayana-lra/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+quarkus.http.test-port=8081
+quarkus.http.host=${HOST:localhost}
+quarkus.ssl.native=true
```

Comment:
Should we automatically enabled SSL in the LRA extension?

Reply:
I will remove it.